### DISCUSSION TITLE
Progressivestun not replaced in some places

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14920] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Beat cop increases the applied affliction strength of `afflictionidentifiers="stun,incrementalstun,progressivestun"`. Notably, progressivestun has been replaced with electricshock, and so beat cop doesn't increase the speed at which e.g. stun batons stun the target.

Additionally, Glass Jaw still gives resistance for `resistancefor="stun,incrementalstun,progressivestun"`, so it doesn't make you more susceptible to electric shock (not really a big deal but I thought I'd mentioned it here)

### Reproduction steps

1. Compare stun baton with and without the beat cop talent; observe that they are the same

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in the latest unstable version now, fix coming up in the next release!
